src/Proxy/LoggerProxy.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Yiisoft\Yii\Debug\Proxy;
+
+use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel;
+use Yiisoft\Yii\Debug\Collector\CollectorInterface;
+
+final class LoggerProxy implements LoggerInterface
```

Comment:
How about structuring classes by their use-case instead of by their type?

```
src
  Collector
      Log
          LogCollector
          LoggerProxy 
       Event
          EventCollector
          EventDispatcherProxy
```

Reply:
I think no, because it means that Proxy may be configured by user's classes.

Reply:
That's fine. I'm not proposing to remove proxies. I'm proposing to group related classes together instead of putting them all over the place grouping by class type.

Reply:
I am about that Proxies potentially may be reused with other collectors, not only for local collector (Log or Event).
And what about collectors that haven't Proxy, it should be placed into folder too?

Reply:
> And what about collectors that haven't Proxy, it should be placed into folder too?

Yes. I think so.

Reply:
What about Proxies without Collectors?

Reply:
These could be moved one way up namespaces.

```
src
  Collector <---- HERE
      Log
          LogCollector
          LoggerProxy 
       Event
          EventCollector
          EventDispatcherProxy
```